site/en/docs/extensions/mv3/getstarted/tut-tabs-manager/index.md

```diff
@@ -0,0 +1,382 @@
+
+```json
+{
+  "manifest_version": 3,
+  "name": "Tab Manager for Google Dev Docs",
+  "version": "1.0",
+  "icons": {
+    "16": "images/icon-16.png",
+    "32": "images/icon-32.png",
+    "48": "images/icon-48.png",
+    "128": "images/icon-128.png"
+  },
+  ...
+}
+```
+
+These manifest keys are explained in more detail in the [Reading time
+tutorial][tut-reading-time-step1]. 
+
+Create an `images` folder and place the icons inside. You can download the icons
+[here][github-tabs-manager-icons].
+
+### Step 2: Create and style the popup {: #step-2 }
+
+The [Action][api-action] API controls the extension action (toolbar icon). When the user clicks on
+the extension action, it will either run some code or open a popup, like in this case. Start by
+declaring the popup in the manifest:
+
+{% Label %}manifest.json:{% endLabel %}
+
+```json
+{
+  ...
+  "action": {
+    "default_popup": "popup.html"
+  },
+  ...
+}
+```
+
+A popup is similar to a web page with one exception: it can't run inline JavaScript. Create a popup
+HTML file and add the following code:
+
+{% Label %}popup.html:{% endLabel %}
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./popup.css" />
+  </head>
+  <body>
+    <template id="li_template">
+      <li>
+        <a>
+          <h3 class="title">Tab Title</h3>
+          <p class="pathname">Tab Pathname</p>
+        </a>
+      </li>
+    </template>
+
+    <h1>Google Dev Docs</h1>
+    <button>Group Tabs</button>
+    <ul></ul>
+
+    <script src="./popup.js" type="module"></script>
+  </body>
+</html>
+```
+
+{% Aside %}
+
+💡 **TIP**: You can use [top level await][mdn-top-level] by adding `type="module”` to the script
+tag.
+
+{% endAside %}
+
+Next, let's style the popup. Create a popup CSS file and include the following code:
+
+{% Label %}popup.css:{% endLabel %}
+
+```css
+body {
+  width: 20rem;
+}
+
+ul {
+  list-style-type: none;
+  padding-inline-start: 0;
+  margin: 1rem 0;
+}
+
+li {
+  padding: 0.25rem;
+}
+li:nth-child(odd) {
+  background: #80808030;
+}
+li:nth-child(even) {
+  background: #ffffff;
+}
+
+h3,
+p {
+  margin: 0;
+}
+```
+
+### Step 3: Manage the tabs {: #step-3 }
+
+The [Tabs API][api-tabs] allows an extension to create, query, modify, and rearrange tabs in the
+browser.
+
+#### Request permission {: #narrow }
+
+Many features in the Tabs API do not require any permissions to use. However, in our case, we need
+to declare a permission to access two sensitive properties: the `title` and the `URL`. 
+
+We could request `"tabs"` permission, which would allow us to see the sensitive properties of all
+the browser tabs. However, to protect user privacy, we will request narrow [host
+permissions][doc-match], giving us access _only_ to the title and URL of **specific sites**.
```

Comment:
I'm playing with the idea of including a screenshot of each permission warning in a details component in case some developers are curious, but at the same time won't make this section so bulky.

Reply:
I changed the copy, feel free to make any change